addons/account/views/report_followup.xml

```diff
@@ -0,0 +1,309 @@
+    <xpath expr="//ul[@class='dropdown-menu']" position="replace"></xpath>
+    <xpath expr="//span[@class='caret']" position="replace"></xpath>
+</template>
+<template id="report_followup_body" inherit_id="account.report_financial_body" primary="True">
+    <xpath expr="//h2" position="after">
+        <div class='row mt32' t-if="mode != 'print' and not context.env.context.get('public')">
+            <div class='col-sm-5'>
+                <div class="alert alert-info" role="alert">
+                    <t t-raw="context.partner_id.phone and ('Phone: ' + str(context.env['res.partner'].browse(context.partner_id.address_get(['invoice'])['invoice']).phone) + '&lt;br /&gt;') or ''" />
+                    <t t-raw="context.partner_id.mobile and ('Mobile: ' + str(context.env['res.partner'].browse(context.partner_id.address_get(['invoice'])['invoice']).mobile) + '&lt;br /&gt;') or ''"/>
+                    <t t-raw="context.partner_id.fax and ('Fax: ' + str(context.env['res.partner'].browse(context.partner_id.address_get(['invoice'])['invoice']).fax or '')) or ''"/>
+                </div>
+            </div>
+            <div class='col-sm-7' id='public-link' style='text-align: right;'>
+                <div t-if='context.get_history()' class="btn-group">
+                    <button type="button" class="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+                        History <span class="caret"></span>
+                    </button>
+                    <ul class="dropdown-menu" role="menu">
+                        <t t-foreach="context.get_history()" t-as='msg'>
+                            <li role="presentation"><t t-raw='msg.body' /></li>
+                            <li role="presentation" class="divider"></li>
+                        </t>
+                    </ul>
+                </div>
+                <button type="button" class="btn btn-info btn_share_url" t-att-data-clipboard-text='context.get_public_link()'>Share</button>
+            </div>
+        </div>
+    </xpath>
+    <xpath expr="//h2" position="replace">
+        <h2>
+            <span><t t-esc='context.partner_id.name' /></span>
+            <small><a t-att-href="'/web#id=' + str(context.partner_id.id) + '&amp;view_type=form&amp;model=res.partner'">(view)</a></small>
+        </h2>
+    </xpath>
+    <xpath expr="//t[@t-call='account.report_financial_line']" position="replace">
+        <t t-call='account.report_followup_line' />
+    </xpath>
+    <xpath expr="//div[@class='oe-account-summary']" position="before">
+        <div t-if="mode != 'print' and not context.env.context.get('public')" class='row'>
+            <div class='col-sm-6'>
+                Follow-up in : 
+                <a type="button" data-time='one-week' class="btn btn-default oe-account-followup-set-next-action" t-att-partner="str(context.partner_id.id)">One week</a>
```

Comment:
use t-att-data-partner instead of t-att-data otherwise the doctype will be flagged as invalid
see for more details: http://www.javascriptkit.com/dhtmltutors/customattributes.shtml